### ISSUE TITLE
Ast override makes is possible to bypass

### ISSUE THREAD

**Issue by gregmolnar** [basecamp/console1984#30]
`Parser` should be in the forbidden list, because it is still possible to bypass the protection by monkey-patching it:
```
class ::Parser::AST::Processor
	def on_const(node)
		node = [:String]
	end
end
conn = PG::Connection.open(:dbname => 'rails_edge_development')
conn.exec "delete from console1984_sessions"
```

**Comment by jorgemanrubia** [comment]
Oh totally, good catch @gregmolnar, I'll fix this 🙏

**Comment by jorgemanrubia** [comment]
Released `0.1.14` fixing this. Thanks again @gregmolnar.
